low-level/framework/utils/drive_utils.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+# CORTX Python common library.
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import json
+import shlex
+import subprocess
+
+
+class DriveUtils:
+    """Base class for drive related utility functions."""
+
+    @staticmethod
+    def get_smart_data():
+        """Extract drive SMART test data using smartctl."""
+        response = []
+        lsscsi_cmd = " ".join(["lsscsi", "|", "grep", "disk"])
+        lsscsi_cmd = shlex.split(lsscsi_cmd)
+        lsscsi_response = subprocess.run(
+                    lsscsi_cmd,
+                    stdout=subprocess.PIPE,
+                    stderr=subprocess.PIPE, shell=True)  # pylint: disable=B606
```

Comment:
can you justify need for shell=true , codacy is flagging it as security issue already

Reply:
Removed subprocess and using SimpleProcess/PipedProcess from cortx.utils.process
Done